src/lib/actionUtils/actionUtils.ts

```diff
@@ -31,12 +33,16 @@ export function checkActionAllowed(action: WrappedAction, actionsContext: Action
 export function runAction(
   action: WrappedAction,
   actionsContext: ActionsContext,
-  storeMethods: RunStoreMethods
+  storeMethods: RunStoreMethods,
+  invokedBy: 'shortcut' | 'palette'
```

Comment:
chore: move the union into a dedicated TS type in `types.ts` and then use here.

Reply:
`RunActionSource` ?

Reply:
how about `RunFrom` or `InvokeBy` or `Initiator`

Reply:
resolved @ [`352bf53` (#15)](https://github.com/itaditya/solid-command-palette/pull/15/commits/352bf53be12947b1c618b2233a579c1e9ab72b85)